xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd

```diff
@@ -166,9 +166,10 @@ Rail transport, Roads and Road transport
 		</xsd:annotation>
 		<xsd:complexContent>
 			<xsd:extension base="containmentAggregationStructure">
-				<xsd:sequence>
-					<xsd:element ref="ServicePattern" maxOccurs="unbounded"/>
-				</xsd:sequence>
+				<xsd:choice maxOccurs="unbounded">
```

Comment:
Do elaborate how this is in anyway related...

Reply:
not at oll I guess. I would leave it in for now.